### ISSUE TITLE
Replace `use-react-router-breadcrumbs` with React Router's implementation

### ISSUE THREAD

**Issue by jonkoops** [keycloak/keycloak#19309]
### Description

The Admin UI uses the [`use-react-router-breadcrumbs`](https://github.com/icd2k3/use-react-router-breadcrumbs) package to render breadcrumbs. However newer versions of React Router have since introduced their [own implementation](https://reactrouter.com/en/main/hooks/use-matches#breadcrumbs) for handling breadcrumbs. We should be using the React Router version instead.

Before picking up this issue it would likely make sense to resolve https://github.com/keycloak/keycloak/issues/19308 first.

**Comment by tirth1** [comment]
I am new to this and I am working on this task. The changes I have made is not reflecting when I run keycloak server from testsuite as mentioned in document. Can someone help me?

**Comment by tirth1** [comment]
As per the React router documentation crumb should be inside handle. So should we change all Routes? All is there any other way?

**Comment by jonkoops** [comment]
Yes, this would indeed involve refactoring all of the route definitions to match what is documented in the React Router documentation.

**Comment by tirth1** [comment]
Please assign it to me. I would like to work on it.

**Comment by tirth1** [comment]
Plese assign it to me. I will work on it.

**Comment by tirth1** [comment]
Please assign it to me. I will work on it.

**Comment by jonkoops** [comment]
Done, thanks for taking this one @tirth1.

**Comment by jonkoops** [comment]
Removing the assignee so someone else can pick this up.

**Comment by Xitija** [comment]
Hi! If this issue is still open, I’d love to work on it. I’m fairly new to open source contributions, so I might need some guidance to get started. Also, could you please let me know the expected timeline for completion?

**Comment by edewit** [comment]
@Xitija this has been open for years so I don't think time is an issue
